src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ CHANGELOG
  * Deprecate the `Http\Client\HttpClient` service, use `Psr\Http\Client\ClientInterface` instead
  * Add `stop_worker_on_signals` configuration option to `messenger` to define signals which would stop a worker
  * Add support for `--all` option to clear all cache pools with `cache:pool:clear` command
+ * Add `DomCrawlerAssertionsTrait::assertAnySelectorTextContains(string $selector, string $text)`
+ * Add `DomCrawlerAssertionsTrait::assertAnySelectorTextSame(string $selector, string $text)`
+ * Add `DomCrawlerAssertionsTrait::assertAnySelectorTextNotContains(string $selector, string $text)`
```

Comment:
Those should go in a new `6.4` section. The feature won't make it into 6.3 as the feature freeze was beginning of March (and the stable release is due for the end of the month, i.e. in one week from now)